npm/private/npm_import.bzl

```diff
@@ -229,19 +226,18 @@ def npm_link_imported_package_store(
         name = "{{}}/dir".format(name),
         srcs = [":{{}}".format(name)],
         output_group = _utils.package_directory_output_group,
-        visibility = visibility,
+        visibility = {link_visibility},
         tags = ["manual"],
     )
 
-    return ":{{}}".format(name)
+    return [":{{}}".format(name)] if {public_visibility} else []
```

Comment:
Oh, this is the spot with the `if True`. I guess if this is a template then it's a bit of a hassle to avoid generating `if True else []`?